package.json

```diff
@@ -38,7 +38,7 @@
     "test": "jest",
     "test:coverage": "jest --coverage",
     "test:watch": "jest --watchAll",
-    "update": "node ./scripts/update-rules.js"
+    "update": "node ./scripts/update-rules.js && node ./scripts/list-jquery-methods.js"
```

Comment:
I took your idea of using a list of jquery methods and ran with it 👟 

Based on https://github.com/wikimedia/eslint-plugin-no-jquery/blob/351eca834e002a056f71072af412bb19255157ce/tools/build-all-methods.js, it uses two new `devDependencies` `jquery` and `jsdom` (already included with `jest`) to build a list of jquery methods that can be used at runtime.

I guess I should credit them too?..

Reply:
Very cool! Feel free to add a comment at the top of the script to link to the code it's based on.